packages/client/hmi-client/src/components/dataset/tera-dataset-jupyter-panel.vue

```diff
@@ -85,23 +85,28 @@
 			</div>
 		</div>
 
-		<!-- Jupyter Chat -->
-		<tera-jupyter-chat
-			ref="chat"
-			:show-jupyter-settings="true"
-			:show-chat-thoughts="props.showChatThoughts"
-			:jupyter-session="jupyterSession"
-			:kernel-status="kernelStatus"
-			:language="selectedLanguage"
-			:default-preview="defaultPreview"
-			@update-kernel-state="(e) => emit('update-kernel-state', e)"
-			@update-kernel-status="updateKernelStatus"
-			@new-dataset-saved="onNewDatasetSaved"
-			@download-response="onDownloadResponse"
-			@update-language="(lang) => emit('update-language', lang)"
-			@update-selected-outputs="(outputs) => emit('update-selected-outputs', outputs)"
-			:notebook-session="props.notebookSession"
-		/>
+		<div>
+			<div v-for="(data, indx) in dataContextDescription" :key="indx">
+				<input class="context-description" :value="data" disabled type="text" />
+			</div>
```

Comment:
Any particular reason why these are disabled `<input>` elements? Are we trying for some specific effect?

Reply:
no deeper meaning aside from `disabled` background colour and grab any rules we have on this that I may have missed

Reply:
Maybe an unordered-list is better at conveying what the code is trying to do. The background colour is like `var(--surface-100)` or something similar to that.

Reply:
Are we really shortening `index` with `indx` to save one character?

Reply:
No we are shortening it to idx to save two characters :P